metafacture-runner/src/main/dist/examples/gnd/crisscross-connections/extract.fix

```diff
@@ -0,0 +1,216 @@
+do put_macro("gndPersonCombinedLabel") # in contrast to morph this is not normalizing utf 8
```

Comment:
Related to: https://github.com/metafacture/metafacture-core/pull/662#discussion_r1941453709